_studio/mfx_lib/encode_hw/mpeg2/src/mfx_mpeg2_encode_full_hw.cpp

```diff
@@ -51,8 +51,7 @@ mfxStatus RunSeqHeader(mfxVideoParam *par, mfxU8* pBuffer, mfxU16 maxLen, mfxU16
     
     mfxU32 bitrate      = 0;
     mfxU32 buffer_size  = 0;
-
-    if (par->mfx.RateControlMethod != MFX_RATECONTROL_CQP) 
+    if ((par->mfx.RateControlMethod != MFX_RATECONTROL_CQP)||(bCqpHrdMode == true))
```

Comment:
This mode requires TargetKbps, but it is not checked to be !0. Does 0 makes sense?